modules/rostests/apitests/appshim/versionlie.c

```diff
@@ -406,4 +410,7 @@ START_TEST(versionlie)
     run_test("Win7SP1VersionLie", &g_Win7SP1);    /* ReactOS specific. Windows does not have this version lie */
     run_test("Win8RTMVersionLie", &g_Win8RTM);
     run_test("Win81RTMVersionLie", &g_Win81RTM);
+    run_test("Win10RTMVersionLie", &g_Win10RTM);    /* ReactOS specific. Windows does not have this version lie */
+    run_test("Win2k16RTMVersionLie", &g_Win2K16RTM);    /* ReactOS specific. Windows does not have this version lie */
```

Comment:
`g_Win2K16RTM` should be `g_Win2k16RTM` (lowercase `k`)

Reply:
Fixed. It was my typo.
Now the checks should succeed.